README.md

```diff
@@ -17,7 +17,8 @@ Maven for building Eclipse bundles and features.
   this somewhere on your file system.
 
 1. [Eclipse 4.5 (Mars) or later](https://www.eclipse.org/downloads/eclipse-packages/). 
-  It's easiest to use the _Eclipse IDE for Java EE Developers_ package.
+  It's easiest to use the _Eclipse IDE for Java EE Developers_ package. You can use
+  Eclipse 4.6 (Neon) or 4.7 (Oxygen) as we define a target platform to build against. 
```

Comment:
You can develop using any version of Eclipse >= what is in your target platform.